installer.gradle

```diff
@@ -24,5 +26,21 @@ task installerBuild(dependsOn: [installerClean], type: Copy) {
   doLast {
     setXmlValue("${buildDir}/installer/buildWar.xml", "version", version)
     file("${buildDir}/installer/files/${archivesBaseName}-${version}.war").renameTo(file("build/installer/files/ODKAggregate.war"))
+
+    if (installBuilderHome != null) {
+      run("mkdir ${buildDir}/installers")
+      setXmlValue("${buildDir}/installer/buildWar.xml", "outputDirectory", "${buildDir}/installers")
+
+      [["linux-x64", "Linux-x64.run"],
+       ["linux", "Linux.run"],
+       ["windows", "Windows.exe"],
+       ["osx", "macOs.app"]
+      ].forEach({ pair ->
+        setXmlValue("${buildDir}/installer/buildWar.xml", "installerFilename", "ODK-Aggregate-${version}-${pair[1]}")
+        run("${installBuilderHome}/bin/builder build ${buildDir}/installer/buildWar.xml ${pair[0]}")
+        run("zip ODK-Aggregate-${version}-${pair[1]}.zip ODK-Aggregate-${version}-${pair[1]}", "${buildDir}/installers")
```

Comment:
Using this PR, I get a very small Mac app. 
```
 35M ODK-Aggregate-v2.0.2-29-gefab50dc-Linux-x64.run.zip
 35M ODK-Aggregate-v2.0.2-29-gefab50dc-Linux.run.zip
 35M ODK-Aggregate-v2.0.2-29-gefab50dc-Windows.exe.zip
238B ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app.zip
```

When I give the buildWar.xml to InstallBuilder directly, then I get a full-sized app.
```
/usr/local/installbuilder/bin/builder build build/installer/buildWar.xml osx
```

My guess is that on macOS, an app isn't a flat-file, it's a directory. 
```
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app/Contents
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app/Contents/MacOS
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app/Contents/MacOS/installbuilder.sh
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app/Contents/MacOS/osx-intel
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app/Contents/Resources
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app/Contents/Resources/installbuilder
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app/Contents/Resources/installbuilder.icns
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app/Contents/Resources/osx-intel.dat
ODK-Aggregate-v2.0.2-29-gefab50dc-macOs.app/Contents/Info.plist
```

So you should probably add a `-r` flag to the zip.
```
run("zip -r ODK-Aggregate-${version}-${pair[1]}.zip ODK-Aggregate-${version}-${pair[1]}", "${buildDir}/installers")
```